src/Extend/Notification.php

```diff
@@ -25,6 +34,19 @@ public function type(string $blueprint, string $serializer, array $channelsEnabl
         return $this;
     }
 
+    /**
+     * @param string $driverName The name of the notification driver.
+     * @param string $driver The ::class attribute of the driver class.
+     *                       This driver should implement \Flarum\Notification\Driver\NotificationDriverInterface.
+     * @return self
+     */
+    public function driver(string $driverName, string $driver)
```

Comment:
Lets add another (optional) argument for "types enabled by default", that takes string names of blueprint classes?

Reply:
ah yes, forgot about this.